guardian/prosemirror-invisibles
# Bump some packages to resolve vulns, then run formatter

## What does this change?
This bumps some packages to resolve some vulnerabilities. I then ran the linter to apply any new rules, and committed some formatting changes. Any changes in this PR outside of 'package.json' and 'yarn.lock' are linting changes.

## How to test

Run some of the scripts in the package.json file locally, for example:
- `yarn watch:serve` to run locally via Vite
- `yarn format` to run prettier
- `yarn build` to try building the app

If these are all working, we should be fairly confident that the version bumps haven't broken anything.
